src/components/ProjectCard/ProjectCard.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import './ProjectCard.scss';
+
+const ProjectCard = ({ project }) => {
+  const { palettes } = project;
+  const paletteCards = palettes.map(palette => {
+    return <section 
+      key={palette.id}
+      className='palette'
+      >
+      <h4>{palette.name}</h4>
+      <div className='five-colors'>
```

Comment:
I think the divs for the colors themselves is great, but we might be able to use something like <section> or something semantic for the .five-colors element if you think it's applicable.